pkgs/leak_tracker_flutter_testing/lib/src/leak_tracking_for_tests.dart

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Should this file be exported by `lib/leak_tracker_flutter_testing.dart` or is it just for package-internal use?